modules/assessment/component/createnewassessments.tsx

```diff
@@ -75,19 +75,20 @@ const CreateTemplate = () => {
         options: [''],
         correct_option: 0,
       },
-    ]);
-    updatelistinobj();
+    ];
+    setList(addquestion);
```

Comment:
Although what you did is fine, you can do this instead and accurately be in sync with the React state for that data

setList((prevList) => ([...prevList, {
        question_no: list.length + 1,
        options: [''],
        correct_option: 0,
      },]));